Utility/Framework/SymfonyFramework.php

```diff
@@ -18,24 +18,6 @@ class SymfonyFramework
     /**
      * @return bool
      */
-    public static function hasDefinitionSharing()
```

Comment:
Looks like I missed this in the `UPGRADE.md` file; it should absolutely be mentioned. As for removal vs depreciation, I believe we should take advantage of not having a `2.x` release yet and remove it.

Without a `2.x` release, those running this development branch should know the API isn't stable. The moment we have an initial `2.0.0` release, this changes and we absolutely need to deprecate, but as of now, we should take the ability to remove outright and do so over depreciation, IMHO.